deploy/iso/minikube-iso/configs/minikube_aarch64_defconfig

```diff
@@ -3,6 +3,7 @@ BR2_aarch64=y
 
 # Toolchain
 BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
+BR2_TOOLCHAIN_BUILDROOT_CXX=y
```

Comment:
was enabling c++ libraries needed ?

Reply:
In menuconfig, under `Target Packages -> Security`, you'll find that you can't enable AppArmor until you meet some prerequisites, like enabling C++.

![image](https://github.com/kubernetes/minikube/assets/108694352/eda3020f-300d-47e6-af2d-289a34cc5c14)